red_october/red_october.py

```diff
@@ -350,16 +350,17 @@ def call(self, endpoint, method='POST', params=None, data=None):
                 success.
         """
         endpoint = '%s/%s' % (self.uri_base, endpoint)
-        if data:
-            data.update({
-                'Name': self.name,
-                'Password': self.password,
-            })
+        if data is None:
```

Comment:
We have a lint check to prevent this, but here is a little lesson in mutability 😉 :

```
>>> def test(key, val, map={}):
...     map[key] = val
...     print map
... 
>>> test('a', 'a-val')
{'a': 'a-val'}
>>> test('b', 'b-val')
{'a': 'a-val', 'b': 'b-val'}
```